### PR TITLE
Bump ruby to 3.0.2

### PR DESCRIPTION
**Pull request by thisisdano**
This possibly addresses an local installation issue I was having with `liquid-c` similar to https://github.com/Shopify/liquid-c/issues/164

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.6.3
\ No newline at end of file
+3.0.2
```
Comment (thisisdano) [inline .ruby-version:1]: Updates Ruby to 3.0.2 since `liquid-c` was failing on my machine with Ruby 2.x

File: Gemfile [hunk 2]
```diff
@@ -11,3 +11,5 @@ gem 'html-proofer'
 
 gem 'rspec-core'
 gem 'rspec-expectations'
+
+gem "webrick", "~> 1.7"
```
Comment (thisisdano) [inline Gemfile:15]: You need to specify `webrick` as a dependency in Ruby 3

**Comment by thisisdano** [comment]
Note that this is not pulling from the github api properly yet. See the [dev docs page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/dw-bump-ruby/documentation/developers/) for an example

File: _plugins/jekyll_get.rb [hunk 3]
```diff
@@ -35,12 +35,12 @@ def load_json(site, d)
       if not File.exists?(path)
         FileUtils.mkpath File.dirname(path)
         print "Caching #{url} in #{path}...\n"
-        data = JSON.load(open(get_final_url(url)))
-        open(path, 'wb') do |file|
+        data = JSON.load(URI.open(get_final_url(url)))
```
Comment (thisisdano) [inline _plugins/jekyll_get.rb:38]: Needs a new syntax in Ruby 3 `open()` → `URI.open()`

File: .ruby-version (hunk 1 above)
Comment (mejiaj) [reply .ruby-version:1]: Should we bump up `.bundler-version` too? Last update was 2 years ago and latest is `2.2.0`.

https://github.com/uswds/uswds-site/blob/b961cbd6f99ff63e16ed450631c051f5b152fc84/.bundler-version#L1

**Review by mejiaj — APPROVED** [review]

**Comment by thisisdano** [comment]
Closed in favor of https://github.com/uswds/uswds-site/pull/1362

**Pull request closed without merging**